i18n/en.json

```diff
@@ -2545,6 +2545,9 @@
   "combined_system_message.removed_from_team.one_you": "You were **removed from the team**.",
   "combined_system_message.removed_from_team.two": "{firstUser} and {secondUser} were **removed from the team**.",
   "combined_system_message.you": "You",
+  "commercial_support.download_support_packet": "Download Support Packet",
```

Comment:
commercial_support.description missing

Reply:
Thanks for that catch. Seems that if I didn't pass the string directly (as oppose to storing it in a variable and passing it), the `make i18n-extract` wouldn't pick it up. Glad you caught it! Thanks :)